src/o365/graph/commands/schemaextension/schemaextension-set.ts

```diff
@@ -0,0 +1,246 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from '../../../base/GraphCommand';
+import Utils from '../../../../Utils';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  description: string;
+  id: string;
+  owner: string;
+  status: string;
+  properties: string;
+  targetTypes: string;
+}
+
+class GraphSchemaExtensionSet extends GraphCommand {
+  public get name(): string {
+    return `${commands.SCHEMAEXTENSION_SET}`;
+  }
+
+  public get description(): string {
+    return 'Updates a Microsoft Graph schema extension';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    if (this.verbose) {
+      cmd.log(`Updating schema extension with id '${args.options.id}'...`);
+    }
+
+    // The default request body always contain owner
+    const body = {
+      owner: args.options.owner
+    };
+
+    // Add the description to request body if any
+    if (args.options.description) {
+      if (this.verbose) {
+        cmd.log(`Will update description to '${args.options.description}'...`);
+      }
+      (body as any).description = args.options.description;
+    }
+
+    // Add the status to request body if any
+    if (args.options.status) {
+      if (this.verbose) {
+        cmd.log(`Will update status to '${args.options.status}'...`);
+      }
+      (body as any).status = args.options.status;
+    }
+
+    // Add the target types to request body if any
+    const targetTypes: string[] = args.options.targetTypes
+      ? args.options.targetTypes.split(',').map(t => t.trim())
+      : [];
+    if (targetTypes) {
```

Comment:
Won't this comparison be always true or will an empty array return false?

Reply:
My bad :) incomplete if... and I did not notice it.
 BTW, empty array are not falsy but `[] == 0` returns true because empty array is converted to empty strict in non strict comparison